planning/scenario_planning/scenario_selector/src/scenario_selector_node/scenario_selector_node.cpp

```diff
@@ -33,15 +31,15 @@
 namespace
 {
 template <class T>
-void onData(const T & data, T * buffer)
+void onData(const T data, T * buffer)
```

Comment:
Ah my rationale was that since other subscription callbacks have removed the explicit `&` when using `ConstSharedPtr` I would keep the same structure here. But you have a good point and since this is not like other subscription callbacks it should be explicit. I have made the fix here: fa49bfd